azure-iot-device/azure/iot/device/iothub/auth/iotedge_authentication_provider.py

```diff
@@ -116,7 +116,7 @@ def get_trust_bundle(self):
         r = requests.get(
             self.workload_uri + "trust-bundle",
             params={"api-version": self.api_version},
-            headers={"User-Agent": urllib.parse.quote_plus(constant.USER_AGENT)},
+            headers={"User-Agent": urllib.parse.quote_plus(ProductInfo.get_iothub_user_agent())},
```

Comment:
>headers={"User-Agent": urllib.parse.quote_plus(ProductInfo.get_iothub_user_agent())}, [](start = 12, length = 85)

I don't think we need this.  This request only goes to the edge daemon (iotedged), and it never goes over any wires.  Does iotedged need any User-Agent at all?

Reply:
I am not sure from the functional perspective of this. This code was already there. I am guessing there are unit tests also , so i have to be a little careful in removing this....but we can check his from Jelani

---
In reply to: [384155183](https://github.com/Azure/azure-iot-sdk-python/pull/464#discussion_r384155183) [](ancestors = 384155183)